docs/source/configuring-slack-for-spidermon.rst

```diff
@@ -0,0 +1,31 @@
+.. _steps:
+
+==========================================
+Guide to configure Slack bot for Spidermon
+==========================================
+
+What are bots?
+==============
+
+A bot is a type of Slack App designed to interact with users via conversation.
+
+To work with Slack Actions, you will need a Slack bot which would send notification to your Slack workplace from Spidermon.
```

Comment:
I think it would be great to link [Slack actions](https://spidermon.readthedocs.io/en/latest/actions.html#slack-action) to the corresponding Spidermon topic. Also, you might want to mention [Slack notifications](https://spidermon.readthedocs.io/en/latest/getting-started.html#slack-notifications) as well.